docs/ref/tran/tls.md

```
@@ -0,0 +1,77 @@
+
+#### Forcing IPv4 or IPv6
+
+To force either IPv4 or IPv6, the scheme may be specified as `tls+tcp4://` or
+`tls+tcp6://`.
+This should only be needed when a hostname might resolve to either address family.
+
+> [!NOTE]
+> Specifying `tls+tcp6://` may not prevent IPv4 hosts from being used with
+> IPv4-in-IPv6 addresses, particularly when listening on wildcard addresses.
+> The details vary across operating systems.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)